autogpt_platform/backend/backend/integrations/webhooks/__init__.py

```diff
@@ -15,6 +16,7 @@
         CompassWebhookManager,
         GithubWebhooksManager,
         Slant3DWebhooksManager,
```

Comment:
Does simple webhook manager like Slant3DWebhooksManager still be needed if we already have the generic one?

Reply:
but idk maybe having it wiht custom outputs would be cool so we get typed data out